### PR TITLE
#6 : Added JS annotations and fixed lint issues for telegram data

### LINKED ISSUE #6: Fix identifier names lint issues
While looking through the code I found multiple lint issues related to identifier names which can be fixed with `JS("<value>")` annotation while keeping dart style, e.g.:

**Current**

```dart
@JS()
class WebAppUser {
  external int get id;
  external bool get is_bot;
  external bool get is_premium;
  external String get first_name;
  external String get last_name;
  external String get username;
  external String get photo_url;
  external String get language_code;
}
```

**Proposal**

```dart
@JS("WebAppUser")
class WebAppUser {
  @JS("id")
  external int get id;
  
  @JS("is_bot")
  external bool get isBot;

  @JS("is_premium")
  external bool get isPremium;
  
  @JS("first_name")
  external String get firstName;

  @JS("last_name")
  external String get lastName;

  @JS("username")
  external String get username;

  @JS("photo_url")
  external String get photoUrl;

  @JS("language_code")
  external String get languageCode;
}
```

I'm opening this issue just to keep track of it. I can do the improvements @khamidjon-khamidov

### PR DESCRIPTION
**Pull request by OlegNovosad**
Closes #6

### CODE REVIEW

File: lib/src/telegram/data/main_button_params.dart [hunk 1]
```diff
@@ -3,11 +3,25 @@ import 'package:js/js.dart';
 @JS()
 @anonymous
 class MainButtonParams {
```
Comment (khamidjon-khamidov) [inline lib/src/telegram/data/main_button_params.dart:5]: @OlegNovosad these variables are in camel case in js: https://core.telegram.org/bots/webapps#mainbutton

Comment (OlegNovosad) [reply lib/src/telegram/data/main_button_params.dart:5]: Fixed. Then it was not working before as well. Thanks for pointing out!

File: lib/src/telegram/data/theme_params.dart [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+  @JS("header_text_color")
+  external String get accentTextColor;
+
+  @JS("section_bg_color")
+  external String get sectionBgColor;
+
+  @JS("section_header_text_color")
+  external String get sectionHeaderTextColor;
+
+  @JS("section_text_color")
```
Comment (khamidjon-khamidov) [inline lib/src/telegram/data/theme_params.dart:43]: this should be `subtitle_text_color`

File: lib/src/telegram/data/theme_params.dart [hunk 3]
```diff
@@ -1,26 +1,49 @@
...
+  @JS("button_text_color")
+  external String get buttonTextColor;
+
+  @JS("primary_bg_color")
+  external String get secondaryBgColor;
+
+  @JS("header_bg_color")
+  external String get headerBgColor;
+
+  @JS("header_text_color")
```
Comment (khamidjon-khamidov) [inline lib/src/telegram/data/theme_params.dart:34]: this should be `accent_text_color`

File: lib/src/telegram/data/theme_params.dart [hunk 4]
```diff
@@ -1,26 +1,49 @@
...
-  external String get section_bg_color;
-  external String get section_header_text_color;
-  external String get subtitle_text_color;
+  @JS("button_color")
+  external String get buttonColor;
+
+  @JS("button_text_color")
+  external String get buttonTextColor;
+
+  @JS("primary_bg_color")
```
[the remark this answers is not in the document]
Comment (OlegNovosad) [reply lib/src/telegram/data/theme_params.dart]: Good catch. Done.

File: lib/src/telegram/data/theme_params.dart (hunk 3 above)
Comment (OlegNovosad) [reply lib/src/telegram/data/theme_params.dart]: Good catch. Done.

File: lib/src/telegram/data/theme_params.dart (hunk 2 above)
Comment (OlegNovosad) [reply lib/src/telegram/data/theme_params.dart]: Good catch. Done.

**Review by khamidjon-khamidov — APPROVED** [review]
@OlegNovosad Thanks for contributing 🙂

**Pull request merged**